sherpa/ui/tests/test_plot_pvalue.py

```diff
@@ -17,33 +17,29 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+"""This is labelled ui but really tests astro/ui
+
```

Comment:
So, should the test file be moved?

Reply:
Possibly but I don't have the time/energy to spend on this compared to the other work we have.